### DISCUSSION TITLE
Playfield border for osu!catch

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#30030] [Q&A]
Self-explainatory. Would work like it does in !Std, but for !CtB.
This also wouldn't need that much work, since a cornered playfield border is already available in the !CtB editor, so there would only need to be added a whole one.

Here's how it looks in the editor: 
![{F265D3F3-2C4F-420C-A26B-49207C11917B}](https://github.com/user-attachments/assets/9b519812-cfba-4081-947b-9e2febd6aef3)

> Why should this be added?

As a person who also plays !CtB, sometimes when there are objects on the edge of the playfield, i don't know how i can play around those edges because i don't know if there is a wall close or not (Because the walls are invisible in case you forgot). If i knew where there is a wall, i would know if i can safely dash on full force or just go for the wall, rather if i have to guess for how long can i still go on the sides. Also it would look cool IMO.

**Comment by peppy** [comment]
Adding this is relatively simple (unless we want to move it over to a skinnable component, which is arguable something we'd eventually want). But the question is: should this work like this osu! playfield border and include the size of the fruit themselves as padding?

**Reply by Tenexxt** [reply]
Visually it would make sense for it to work like osu! Since the platter is scaled with CS, I think it should be fine to a certain degree

>(unless we want to move it over to a skinnable component, which is arguable something we'd eventually want).

I personally could deal with it not being skinnable, but i think there could be very different opinions from the rest of the players, so I think looking for feedback on that would be better (after of course you'd settle that if people will want it you can add it, or you can just not say anything about and not add skinning )

**Comment by bdach** [comment]
Why would the border be cornered though? Shouldn't it just be two lines on the sides?

**Reply by Tenexxt** [reply]
I forgot that (on argon atleast) there is a line with the catcher lol, so there could only be lines at the sides like you said.
Also it would look better since It would look kinda awkward ngl.
